version.c

```diff
@@ -393,9 +379,9 @@ void print_version(void)
 #endif
 
 #ifdef HAVE_IDN2_H
-  printf("\nlibidn: %s (compiled with %s)", idn2_check_version(NULL), IDN2_VERSION);
+  printf("%s\n",mutt_idna_print_version());
 #elif defined(HAVE_LIBIDN)
-  printf("\nlibidn: %s (compiled with %s)", stringprep_check_version(NULL), STRINGPREP_VERSION);
+  printf("%s\n",mutt_idna_print_version());
 #endif
```

Comment:
Both versions of IDN define the symbol `HAVE_LIBIDN`, so we only need the second case, here.